packages/manager/modules/anthos/src/dashboard/components/assign-private-ip/constant.js

```diff
@@ -0,0 +1,12 @@
+export const IPV4_REGEX = {
+  CIDR_RANGE_32: new RegExp(
+    '^([0-9]{1,3}\\.){3}[0-9]{1,3}(\\/([0-9]|[1-2][0-9]|3[0-2]))?$',
+  ),
+  PRIVATE_IP: new RegExp(
+    '^(127(?:\\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$)|(10(?:\\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$)|(192\\.168(?:\\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){2}$)|(172\\.(?:1[6-9]|2\\d|3[0-1])(?:\\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){2}$)',
```

Comment:
Could you please document the source of the regex please?

Reply:
it's comes regarding this:
https://datatracker.ietf.org/doc/html/rfc1918